modules/exploits/linux/http/craftcms_preauth_rce_cve_2025_32432.rb

```diff
@@ -0,0 +1,203 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Payload::Php
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Craft CMS Preauth RCE Meterpreter (CVE-2025-32432)',
```

Comment:
```suggestion
        'Name' => 'Craft CMS Preauth RCE (CVE-2025-32432)',
```

The module name usually matches the affected functionality; ie, something like:

```suggestion
        'Name' => 'Craft CMS Image Transform Preauth RCE (CVE-2025-32432)',
```

Reply:
Updated